categories_map.py

```diff
@@ -1292,6 +1293,9 @@
     "RecoPixelVertexing/PixelTrackFitting",
     "RecoPixelVertexing/PixelTriplets",
     "RecoPixelVertexing/PixelVertexFinding",
+    "RecoPPS/Configuration",
+    "RecoPPS/PixelLocal",
```

Comment:
@perrotta , I guss this should be `RecoPPS/Local` correct?

Reply:
Ops, yes. My fault sorry.
Thank you for noticing @smuzaffar : I prepare a fix

Reply:
See https://github.com/cms-sw/cms-bot/pull/1272